### PR TITLE
Fix syncPoolHandler not passing limit config to syncUserHandler

### PR DESCRIPTION
**Pull request by mostlygeek**
Stupid bug where syncPoolHandler did not pass the limit config to syncUserHandler. 

@grigoryk review please?

### CODE REVIEW

File: web/syncPoolHandler_elements.go [hunk 1]
```diff
@@ -155,7 +157,7 @@ func (p *handlerPool) getElement(uid string) (*poolElement, bool, error) {
 
 		element = &poolElement{
 			uid:     uid,
-			handler: NewSyncUserHandler(uid, db, nil),
```
Comment (grigoryk) [inline web/syncPoolHandler_elements.go:158]: oops!

**Review by grigoryk — APPROVED** [review]
Change seems to make sense to me, with a caveat that I don't know this code.

File: web/syncPoolHandler_elements.go (hunk 1 above)
Comment (grigoryk) [reply web/syncPoolHandler_elements.go:158]: I guess this nil causes a default config to be created/picked up? Which implies that there are some defaults somewhere, and they're not what you actually intend to use?

Comment (mostlygeek) [reply web/syncPoolHandler_elements.go:158]: Yup. If it is `nil` it will use the default config. I should actually change it to match the pysync server's

**Pull request merged**